sound/soc/intel/boards/bxt_tdf8532.c

```diff
@@ -205,6 +205,18 @@ static struct snd_soc_dai_link broxton_tdf8532_dais[] = {
 		.dpcm_playback = 1,
 		.no_pcm = 1,
 	},
+	{
+		/* SSP4 for vm */
```

Comment:
Why do we need this ? If it's for tmp debug then lets put a big comment here stating it's tmp debug for feature XXX and will be removed.

Reply:
This is a tmp solution for BE link. I will change it to a new solution with the new topology mode later.